### PR TITLE
[SAF-41] fix color constant and background for mobile vs desktop views

### PR DESCRIPTION
**Pull request by luchopcerra**
https://github.com/user-attachments/assets/831895fc-9eed-4968-acc9-f53e4e746934

### CODE REVIEW

File: apps/frontend/src/app/5_pages/AaveReserveOverviewPage/components/BorrowDetailsGraph/components/Chart/Chart.constants.ts [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
 
 export const CUSTOM_CANVAS_BACKGROUND_COLOR = {
   id: 'customCanvasBackgroundColor',
   beforeDraw: (chart, options) => {
     const { ctx } = chart;
+    const windowWidth = window.innerWidth;
     ctx.save();
     ctx.globalCompositeOperation = 'destination-over';
-    ctx.fillStyle = '#1e2128';
+    // Cambia el color según el ancho de la ventana
```
Comment (matzapata) [inline apps/frontend/src/app/5_pages/AaveReserveOverviewPage/components/BorrowDetailsGraph/components/Chart/Chart.constants.ts]: comments in english please

File: apps/frontend/src/app/5_pages/AaveReserveOverviewPage/components/InterestRateModelGraph/components/Chart/Chart.constants.ts [hunk 2]
```diff
@@ -1,21 +1,22 @@
...
 
 export const CUSTOM_CANVAS_BACKGROUND_COLOR = {
   id: 'customCanvasBackgroundColor',
   beforeDraw: (chart, options) => {
     const { ctx } = chart;
+    const windowWidth = window.innerWidth;
     ctx.save();
     ctx.globalCompositeOperation = 'destination-over';
-    ctx.fillStyle = '#1e2128';
+    // Cambia el color según el ancho de la ventana
```
Comment (matzapata) [inline apps/frontend/src/app/5_pages/AaveReserveOverviewPage/components/InterestRateModelGraph/components/Chart/Chart.constants.ts]: same

File: apps/frontend/src/app/5_pages/AaveReserveOverviewPage/components/SupplyDetailsGraph/components/Chart/Chart.constants.ts [hunk 3]
```diff
@@ -1,21 +1,22 @@
...
     ctx.save();
     ctx.globalCompositeOperation = 'destination-over';
-    ctx.fillStyle = '#1e2128';
+    // Cambia el color según el ancho de la ventana
+    if (windowWidth < 640) {
+      // Breakpoint 'sm'
+      ctx.fillStyle = GRAY_90;
+    } else {
+      ctx.fillStyle = GRAY_80;
+    }
```
Comment (matzapata) [inline apps/frontend/src/app/5_pages/AaveReserveOverviewPage/components/SupplyDetailsGraph/components/Chart/Chart.constants.ts]: Translations. But also here and for all the rest, if you need to access colors better to do it taking the tailwind config programmatically so we have a single source of truth. Sovryn has this package: `@sovryn/tailwindcss-config/index.js` you can see they use it as presset. Something like this should work:

```
import { theme} from "@sovryn/tailwindcss-config"
theme.screens.md
theme.colors['gray-90']
```

File: apps/frontend/src/app/5_pages/AaveReserveOverviewPage/components/SupplyDetailsGraph/SupplyDetailsGraph.tsx [hunk 4]
```diff
@@ -9,13 +9,15 @@ import { StatisticsCard } from '../../../../2_molecules/StatisticsCard/Statistic
...
 import { Chart } from './components/Chart/Chart';
-import { harcodedData, LINE_COLOR } from './components/Chart/Chart.constants';
+import { harcodedData } from './components/Chart/Chart.constants';
 import { MockData } from './components/Chart/Chart.types';
 
 const pageTranslations = translations.aaveReserveOverviewPage.supplyDetails;
 
 type SupplyDetailsGraphProps = {};
 
+const LINE_COLOR = '#f58c31';
```
Comment (matzapata) [inline apps/frontend/src/app/5_pages/AaveReserveOverviewPage/components/SupplyDetailsGraph/SupplyDetailsGraph.tsx]: As per figma seems that it's also a color from the pallete. So better use from there. This is `primary-30` and the green is `positive`. You can use the import as described other comment

File: apps/frontend/src/app/5_pages/AaveReserveOverviewPage/components/BorrowDetailsGraph/components/Chart/Chart.constants.ts [hunk 5]
```diff
@@ -1,21 +1,22 @@
...
+    if (windowWidth < 640) {
```
Comment (juandahl) [inline apps/frontend/src/app/5_pages/AaveReserveOverviewPage/components/BorrowDetailsGraph/components/Chart/Chart.constants.ts]: I haven't seen how the project manage breakpoints but isn't there constants for them?

File: apps/frontend/src/app/5_pages/AaveReserveOverviewPage/components/BorrowDetailsGraph/components/Chart/Chart.tsx [hunk 6]
```diff
@@ -110,7 +108,7 @@ export const Chart: FC<ChartProps> = ({ mockData }) => {
       onClick={e => {
         e.stopPropagation();
       }}
-      className="w-full rounded"
+      className={`lg:p-6 lg:bg-gray-80`}
```
Comment (juandahl) [inline apps/frontend/src/app/5_pages/AaveReserveOverviewPage/components/BorrowDetailsGraph/components/Chart/Chart.tsx]:
```suggestion
      className="lg:p-6 lg:bg-gray-80"
```

File: apps/frontend/src/app/5_pages/AaveReserveOverviewPage/components/SupplyDetailsGraph/SupplyDetailsGraph.tsx (hunk 4 above)
Comment (juandahl) [reply apps/frontend/src/app/5_pages/AaveReserveOverviewPage/components/SupplyDetailsGraph/SupplyDetailsGraph.tsx]: I aggree! I think we should  only hardcode those colors which are "border cases". 
We could also add some new colors to the palette if they appears many times as well.

**Review by juandahl — APPROVED** [review]
LGTM. Just minor comments

**Pull request merged**